send_to_telemetry.py

```diff
@@ -0,0 +1,54 @@
+from dataclasses import asdict, dataclass
+import json
+import sys
+
+
+@dataclass
+class TestResult:
+    appName: str
+    channel: str
+    version: str
+    url: str
+    metrics: dict
+
+
+def main(path):
+    with open(path) as f:
+        data = json.load(f)
+
+    with open("wpt.json") as f:
+        metrics = json.load(f)
+
+    for test in data:
+        sample = test["data"]["median"]["firstView"]
+        metrics = {
+            "TTFB": sample.TTFB,
+            "startRender": sample.render,
```

Comment:
This is the only one that doesn't match the name, could we submit "render" as the key instead? That would allow us to just have a list, or even use the keys from `metrics.json`.

Reply:
Fixed; it would be great to use the keys in a follow-up PR.

Reply:
Got them working in https://github.com/mozilla/wpt-api/pull/236/commits/6ef5ab1c29fdb699018f95559ace05a541446271 and then am trying to write them back out for Telemetry, in https://github.com/mozilla/wpt-api/pull/236/commits/cadf9d402cd2904969d83a0c4162c8482b5371bf